src/geo/map.js

```diff
@@ -26,39 +25,43 @@ var Map = Model.extend({
 
   RELOAD_DEBOUNCE_TIME: 10,
 
+  parse: function (r) {
```

Comment:
According to [Backbone docs](http://backbonejs.org/#Model-parse):

> **parse** is called whenever a model's data is returned by the server, in fetch, and save. The function is passed the raw response object, and should return the attributes hash to be set on the model. The default implementation is a no-op, simply passing through the JSON response. Override this if you need to work with a preexisting API, or better namespace your responses.

I'd personally not use it in this case since there's no server response here, and It forces us to always do:

```
var map = new Map({ ... }, { parse: true })
```

This is not very convenient IMO... It's easy to forget. I originally added this code to the constructor but I was probably wrong. We could acutally move it [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/api/vizjson.js#L10) (we'd still need to set original_center, original_view_bounds_sw and original_view_bounds_ne in the `initialize` method). What do you think?